zyte_common_items/base.py

```diff
@@ -118,3 +115,45 @@ def _apply_field_types_to_sub_fields(cls, item: Optional[Dict]):
             item.update({key: cls.from_list(item.get(key, [])) for key, cls in (from_list or {}).items()})
 
         return item
+
+
+@_export_attrs()
+class AdditionalProperty:
+    name: str
+    value: str
+
+
+@_export_attrs(kw_only=True)
+class AggregateRating:
+    best_rating: Optional[float] = None
+    rating_value: Optional[float] = None
+    review_count: Optional[int] = None
+
+
+@_export_attrs()
+class Brand:
+    name: str
+
+
+@_export_attrs()
+class Breadcrumb:
+    name: str
+    link: str
```

Comment:
I think that's fine to have one of the fields absent in Breadcrumbs, i.e. probably both should be declared as Optional